_episodes/04-history.md

```diff
@@ -128,23 +149,26 @@ and "unique" really does mean unique:
 every change to any set of files on any computer
```

Comment:
This sentence is really weird; it makes it sound like the same commit will have a different hash on each computer.